cfbs/main.py

```diff
@@ -53,6 +55,11 @@ def get_args():
         default=None,
         help="Expected checksum of the downloaded file",
     )
+    parser.add_argument(
+        "--keep-order",
+        help="Keep order of items in the JSON in '%s pretty'" % os.path.basename(sys.argv[0]),
```

Comment:
How important is it to change the name of `cfbs` and have it show up differently in the help output?

IMO this is much more readable and greppable: 

```suggestion
        help="Keep order of items in the JSON in 'cfbs pretty'",
```